src/ts/components/subtitleoverlay.ts

```diff
@@ -81,10 +81,55 @@ export class SubtitleOverlay extends Container<SubtitleOverlayConfig> {
     let subtitleManager = new ActiveSubtitleManager();
     this.subtitleManager = subtitleManager;
 
+    let cea608CssClass = this.prefixCss('cea608');
+    let cea608fontSize = 1;
+
+    let generateCea608Ratio = () => {
+      let label = new SubtitleLabel({
+        // One letter label used to calculate the height wifth ratio of the font
```

Comment:
Grammar/typo: `A label used to ... height/width ratio ...` ... `This works because ...`